dataplane/standalone/lucius/lucius.go

```diff
@@ -87,3 +91,8 @@ func initialize(port int) {
 
 func main() {
 }
+
+func init() {
+	// TODO: Figure a better way to config this when used standalone.
+	flag.Set("log_dir", "/var/log/syncd")
```

Comment:
Why is this being done?  It is possible to find which flags were actually set by using flag.Visit.

Reply:
this just temporary until we run the dataplane in its own container (and can set the args)